### PR TITLE
Adds non-working button to congress person show page

### PR DESCRIPTION
**Pull request by TheKaterTot**
This adds a dummy button to the show page for a Congressperson. It currently alternate on click between 'Follow' and 'Unfollow'.

### CODE REVIEW

File: public_officials/functional_tests/test_public_officials.py [hunk 1]
```diff
@@ -47,9 +47,11 @@ def test_checks_for_content_on_official_show(self):
             self.browser.get(self.live_server_url + '/legislators/%d/' % legislator.id)
             name_text = self.browser.find_element_by_tag_name('h1').text
             status_text = self.browser.find_element_by_tag_name('h3').text
+            follow_button_text = self.browser.find_element_by_css_selector('.follow').text
             bio_text = self.browser.find_element_by_css_selector('.profile').text
             stats_text = self.browser.find_element_by_css_selector('.stats').text
             self.assertIn("Diana DeGette", name_text)
+            self.assertIn("Follow", follow_button_text)
```
Comment (meyerhoferc) [inline public_officials/functional_tests/test_public_officials.py]: nice feature test 👍

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 2]
```diff
@@ -117,4 +124,5 @@ $(document).ready(function(){
   getIndustryContributors(legislator_id);
   getSponsoredBills(legislator_pid);
   getVotingHistory(legislator_pid);
+  $('.follow').on('click', followLegislator);
```
Comment (meyerhoferc) [inline public_officials/static/public_officials/js/legislator_detail.js]: 💯 great use of JS & jQuery

File: public_officials/templates/public-officials/detail.html [hunk 3]
```diff
@@ -36,9 +36,12 @@
     </nav>
 
     <div class="container-fluid">
-      <div class="row-well profile col-md-4 col-md-offset-1 legislator" data-legislator-id="{{ legislator.cid }}" data-legislator-pid="{{ legislator.pid }}">
+      <div class="profile col-md-4 col-md-offset-1 legislator" data-legislator-id="{{ legislator.cid }}" data-legislator-pid="{{ legislator.pid }}">
         <h1>{{ legislator.first_name }} {{ legislator.last_name }}</h1>
       </div>
+      <div class="col-md-1 col-md-offset-5">
+        <button class="follow btn">Follow</button>
+      </div>
```
Comment (meyerhoferc) [inline public_officials/templates/public-officials/detail.html]: great idea on the placement here. looks great!

File: public_officials/functional_tests/test_public_officials.py (hunk 1 above)
Comment (andrewdwooten) [inline public_officials/functional_tests/test_public_officials.py]: Looks great. What do we think we'll want to do with the button for an unauthenticated user? Deactivate it or just not display it?

File: public_officials/static/public_officials/js/legislator_detail.js (hunk 2 above)
Comment (andrewdwooten) [inline public_officials/static/public_officials/js/legislator_detail.js]: You made this very easy to read through. Awesome work!

File: public_officials/functional_tests/test_public_officials.py (hunk 1 above)
Comment (TheKaterTot) [inline public_officials/functional_tests/test_public_officials.py]: I think just not show it.

**Pull request merged**